handlers/bulk_refund.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Probably worth moving these xml files into a `test_files` subfolder to separate from the actual code.